x-pack/plugin/core/src/main/java/org/elasticsearch/xpack/core/search/action/TransportOpenPointInTimeAction.java

```diff
@@ -72,6 +72,7 @@ protected void doExecute(Task task, OpenPointInTimeRequest request, ActionListen
             .preference(request.preference())
             .routing(request.routing())
             .allowPartialSearchResults(false);
+        searchRequest.setCcsMinimizeRoundtrips(false);
```

Comment:
Why is it needed ?

Reply:
We need to disable ccs_minimize_roundtrips because we don't know how to [merge](https://github.com/elastic/elasticsearch/blob/4e974bc91e5af4037ab08345a248e980163c4511/server/src/main/java/org/elasticsearch/action/search/SearchResponseMerger.java#L206-L207) a point in time from multiple sub-search requests.